main.go

```diff
@@ -173,7 +172,8 @@ func retryFailedTests(testRunner TestRunner, maxRetries int, timeline *[]api.Tim
 	retries := 0
 	for retries < maxRetries {
 		retries++
-		fmt.Printf("Attempt %d of %d to retry failing tests\n", retries, maxRetries)
+
+		fmt.Printf("+++ Buildkite Test Splitter: ♻️ Attempt %d of %d to retry failing tests\n", retries, maxRetries)
```

Comment:
Persnickety old man in the corner here but how come this one has `+++` at the start but the logErrorAndExit one doesn't?

Reply:
My understanding of `+++` is that the group is automatically expanded? So all the re-run spec tests will automatically be expanded in the job log, rather than having to open the group to view the rerun specs. Whereas I think because the logErrorAndExit log lines are discrete and not part of a group they just get printed anyways? I could be very wrong though

Reply:
Ah I see now. In that case do we want to have another group for the initial run? Or do you think it doesn't matter?

Reply:
Added a group for this, the errors that might occur from the test cmd being wrong will be printed after this group